test/integration/custom-error/next.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  target: 'serverless'
```

Comment:
This should be dual-tested for normal and serverless please!

Reply:
Updated